kafka-rest/src/test/java/io/confluent/kafkarest/resources/v3/ProducerMetricsTest.java

```diff
@@ -78,6 +79,7 @@ public void testRateMetrics() throws Exception {
         .forEach(
             n -> {
               metrics.recordError();
+              metrics.recordRateLimited();
```

Comment:
More of a thought than a requirement.... Is this test not very good..... As in, I